### ISSUE TITLE
Testdev set

### ISSUE THREAD

**Issue by chojw** [gqa-ood/GQA-OOD#6]
Hi, 

I saw in one of your replies that you used the GQA-OOD validation set for Testing,
does this mean that the ood_testdev was not used at all??

Thanks in advance!

I referenced the issue below:

Dear @alice-cool,

In the paper, we did:

- **Training**: GQA balanced training set
- **Validation**: GQA-OOD validation set
- **Test**: GQA-OOD validation set

Corentin K.

_Originally posted by @gqa-ood in https://github.com/gqa-ood/GQA-OOD/issues/4#issuecomment-984491135_
